core/document/src/layers/style/mod.rs

```diff
@@ -56,6 +56,9 @@ impl PathStyle {
 	pub fn new(stroke: Option<Stroke>, fill: Option<Fill>) -> Self {
 		Self { stroke, fill }
 	}
+	pub fn set_fill(&mut self, fill: Option<Fill>) {
```

Comment:
I would suggest set_fill should take fill as an argument and wrap it in Some before assigning.
The value is wrapped in an option because you don't have to have a fill, but if you set the fill, thät is always the case

Reply:
We may want a way to clear the fill on for example a rectangle with both an outline and a fill, but it should probably be with a `clear_fill` method

Reply:
Good points. I had chosen this method to ensure the value could be reset to None. I just implemented @0HyperCube's recommendation, as it's the best of both worlds.